apps/telepathy/telepathy-resiprocate.cxx

```diff
@@ -27,13 +27,18 @@
 #include <rutil/Logger.hxx>
 #include <resip/recon/SipXHelper.hxx>
 
-
+#include "rutil/Errdes.hxx"
 #include "Protocol.hxx"
 
 using namespace tr;
 
 int main(int argc, char *argv[])
 {
+   //Creating error map for OS, SSL and X509 error numbers
+   ErrnoError::CreateMappingErrorMsg();
+   OpenSSLError::CreateMappingErrorMsg();
+   //X509Error::CreateMappingErrorMsg();
```

Comment:
I have not used any X509 Error message in the project, should I remove all the X509 error messages from Errdes.cxx or would it be needed in future ?